src/pages/unauth/signIn/OtpCheck.tsx

```diff
@@ -0,0 +1,53 @@
+import * as S from "./style";
+import { GrSecure } from "react-icons/gr";
+import React, { useContext, useState } from "react";
+import * as auth from "api/auth";
+import { setAuth } from "userAuth";
+import { AuthContext } from "hooks/context/AuthContext";
+import { useNavigate } from "react-router-dom";
+
+export default function OtpCheck(props: { username: string; accessToken: string; }) {
+  const setSigned = useContext(AuthContext);
+  const navigate = useNavigate();
+  const [otpInput, setOtpInput] = useState("");
+  const [noticeFail, setNoticeFail] = useState("");
+
+  function onInputOtpHandler(e: React.ChangeEvent<HTMLInputElement>) {
+    setOtpInput(e.target.value);
+    if (noticeFail) setNoticeFail("");
+  }
+
+  async function postOtpHandler(e: React.MouseEvent<HTMLButtonElement>) {
+    e.preventDefault();
+    if (otpInput) {
+      const res = await auth.checkOtpLogin(otpInput, props.accessToken);
+      if (res && (res.status === 201)) {
+        if (setSigned) setSigned(true);
+        setAuth({
+          username: props.username,
+          token: res.data.accessToken,
+        });
+        navigate("/");
+      } else {
+        console.log(res);
+      }
+      setNoticeFail("인증번호를 확인해주세요.");
```

Comment:
```suggestion
      }
      console.log(res);
      setNoticeFail("인증번호를 확인해주세요.");
```

위의 `if (응답코드 === 201)`의 결과로 페이지 이동되니까 그 이후는 `else` 없어도 될것 같아요~!

Reply:
응답코드 !== 201 인 경우 처리하는 부분인데 setNoticeFail을 else 안으로 옮기겠습니다

Reply:
수정했습니다

Reply:
아 응답코드 !== 201인 경우 이미 위에서 200인 경우가 navigate되어 아래의 코드를 실행하는 경우가 없어서(early return인 셈이죠!) else 조건문이 없어도 될거 같다는 얘기였어요~!